lib/checks/001-deprecations.js

```diff
@@ -21,32 +21,32 @@ checkDeprecations = function checkDeprecations(theme, themePath) {
                 ruleCode: 'GS001-DEPR-IMG'
             },
             {
-                helperRegEx: '{{author.image}}',
+                helperRegEx: new RegExp(/{{author\.image}}/),
```

Comment:
Awesome! Thanks a lot! For the sake of consistency, I changed it now, so every regex has the `g` flag.